rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -1219,18 +1211,18 @@ class ExpressionGenerator extends RosettaExpressionSwitch<JavaStatementBuilder,
 	}
 
 	override protected caseSwitchOperation(SwitchOperation expr, Context context) {
```

Comment:
I think this one needs a bit of thought. Let's discuss tomorrow.

Reply:
Should ignore metadata on the inputs to the switch operations

Reply:
fixed